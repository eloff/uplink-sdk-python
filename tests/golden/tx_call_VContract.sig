r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 36673953526167094003488209290387043875236740401217950133399348588401528687602
