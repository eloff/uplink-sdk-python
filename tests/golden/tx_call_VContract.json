{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                        "tag": "VContract"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zNjY3Mzk1MzUyNjE2NzA5NDAwMzQ4ODIwOTI5MDM4NzA0Mzg3NTIzNjc0MDQwMTIxNzk1MDEzMzM5OTM0ODU4ODQwMTUyODY4NzYwMg=="
}